server/game/cards/12-2T2D/Bedazzle.js

```diff
@@ -0,0 +1,40 @@
+const SpellCard = require('../../spellcard.js');
+
+class Bedazzle extends SpellCard {
+    setupCardAbilities(ability) {
+        this.spellAction({
+            title: 'Bedazzle',
+            playType: 'cheatin resolution',
+            cost: ability.costs.bootSelf(),
+            difficulty: 3,
+            choosePlayer: true,
+            onSuccess: (context) => {
+                context.chosenPlayer.modifyRank(-2, context);
+                this.game.addMessage('{0} uses {1} to lower {2}\'s draw hand by 2 ranks; Current rank is {3}',
+                    context.player, this, context.chosenPlayer, context.chosenPlayer.getTotalRank());
+                if(this.game.shootout) {
+                    context.ability.selectAnotherTarget(context.player, context, {
+
+                        activePromptTitle: 'Choose opposing dude to bedazzle',
+                        waitingPromptTitle: 'Waiting for opponent to select dude',
+                        cardCondition: card => card.location === 'play area' &&
+                            card.controller === context.player.getOpponent() &&
+                            card.isParticipating(),
+                        cardType: 'dude',
+                        onSelect: (p, dazzlee) => {
+                            const dazzleAmount = Math.max(dazzlee.bullets, 0);
+                            ability.effects.modifyBullets(0 - dazzleAmount);
+                            this.game.addMessage('{0} gets -{1} bullets due to the {2}', dazzlee, dazzleAmount, this);

```

Comment:
We have an internal rule that message should always start with a user.

Reply:
Are the seperate messages fine as long as the user is at the start? I was never happy with my attempts to combine them.

Reply:
yeah, it is fine. I am also trying to combine them sometimes, but usually am not happy.